CHANGELOG.md

```diff
@@ -1,3 +1,22 @@
+Changes in 0.6.1 (2016-11-23)
+=============================
+
+New features:
+ - Storing IRC Passwords Encrypted at Rest: No longer will users need to `IDENTIFY` with NickServ! Simply send the admin room command `!storepass [server.name] password`, and your password that you would normally send to NickServ will be stored in the bridge database in an encrypted form (and not as a hash). WARNING: The bridge needs to send plaintext passwords to the IRC server when the bridge is connecting the user to IRC (typically on startup). This means that the key used to decrypt passwords must be loaded into memory on startup, and the key file will typically be stored on the same machine as the bridge itself. `!removepass [server.name]` will remove the password that the user has set from the database.
```

Comment:
Please break up this line to make it easier to read if you don't word wrap, and to make it easier to target line comments.

> Storing IRC Passwords Encrypted at Rest: No longer will users need to `IDENTIFY` with NickServ!

It feels like you've written this for the consumer (user) audience rather than the bridge admin audience so this reads weirdly.

I would keep it specific and boring: "Matrix users can now specify a [server password](https://en.wikipedia.org/wiki/List_of_Internet_Relay_Chat_commands#PASS) to authenticate with the IRC server on startup. On most IRC servers, this is an alternative mechanism to authenticate with `NickServ`. To enable this functionality in the bridge, a private key needs to be generated.

> WARNING: The bridge needs to send plaintext passwords to the IRC server when the bridge is connecting the user to IRC (typically on startup). This means that the key used to decrypt passwords must be loaded into memory on startup, and the key file will typically be stored on the same machine as the bridge itself.

This is confusing and warning about the wrong things. You need to be shouting about the fact that the bridge is forced to send the plaintext password and not a hash of the password to the IRC server, meaning that Matrix users are *trusting the bridge with their actual password*, which is very different to how password systems typically operate (which don't store the actual password, but a hash).


Overall this feature may warrant sub bullet points to break it up:
 - What it is
 - How does a sysadmin set it up
 - How does a matrix user use it
 - Risks involved